pages/index.js

```diff
@@ -1,75 +1,162 @@
+import { FaHtml5 } from "react-icons/fa";
+import { FaCog } from "react-icons/fa";
+import { FaCogs } from "react-icons/fa";
+import { FaUserCog } from "react-icons/fa";
+import { FaProjectDiagram } from "react-icons/fa";
+import { FaChartLine } from "react-icons/fa";
 
 export default function Home() {
+  const Color1 = "borderColor1";
+  const Color2 = "borderColor2";
+  const Color3 = "borderColor3";
   return (
-    <div className={styles.container}>
+    <div className=" ">
       <Head>
-        <title>Create Next App</title>
-
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
+        <title>Jobie</title>
+        <link rel="icon" href="/icon_Logo.ico" />
       </Head>
-      <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-        <p className={styles.description}>
-          Get started by editing{" "}
-          <code className={styles.code}>pages/index.js</code>
-        </p>
-
-        <div className={styles.counter}>
-          <Link href="/counter">
-            <a>Goto Counter Page</a>
-          </Link>
+      <Header
+        title="10,254 Jobs"
+        title2="Are Listed Here!"
+        subtitle="Find your dream job now"
+        light={false}
+        img={HeaderImage}
+      >
+        <SearchButton />
+      </Header>
+      <div className="bg-body px-4 lg:px-48 w-full">
+        <div className="flex  justify-around py-16">
+          <JobTypeCard
+            title="Full-Time Jobs"
+            text="if you're set on you're career and want to work full time, then a full time job is just the one for you, clicking here will show you all the current full time jobs in our platform"
+            borderColor={Color1}
+          />
+          <JobTypeCard
+            title="Part Time Jobs"
+            text="if you're a student or can't spend to much time on work, then a part time job is what you're looking for. click here and we'll show you all the part-time job offers"
+            borderColor={Color2}
+          />
+          <JobTypeCard
+            title="Remote Jobs"
+            text="if you prefer working from home, with no constraints to a specific place then a remote job is a perfect fit and we've got plenty of that, click and let us show you all the available positions"
+            borderColor={Color3}
+          />
         </div>
+        <div className="py-16">
+          <h1 className="text-primary font-semibold text-lg lg:text-2xl xl:text-4xl">
+            {" "}
+            Explore by categories
+          </h1>
+          <div className="grid grid-rows-2 gap-10 py-16">
+            <div className="flex  justify-around">
+              <Categories
+                Logo={FaDraftingCompass}
+                title="Design"
+                subtitle="67 Job Vacancy"
+                link=""
+              />
```

Comment:
align the second row the same as the first with 4 columns. And add the "View all categories" link for the last column. The link just goes to job search page which I am working on. Or put anything else if you have a better idea